### PR TITLE
Task/high order count opti

### PR DESCRIPTION
**Pull request by liampauling**

### CODE REVIEW

File: flumine/markets/blotter.py [hunk 1]
```diff
@@ -70,11 +71,12 @@ def _create_packages(
...
-    def live_orders(self) -> bool:
-        for order in self._orders.values():
-            if order.complete is False or order.trade.complete is False:
-                return True
-        return False
+    def live_orders(self):
+        return iter(self._live_orders)
+
+    @property
+    def has_live_orders(self):
```
Comment (liampauling) [inline flumine/markets/blotter.py:78]: Might rename this

File: flumine/backtest/backtest.py [hunk 2]
```diff
@@ -94,8 +94,10 @@ def _process_market_books(self, event: events.MarketBookEvent) -> None:
 
             # process current orders
             blotter = market.blotter
-            for order in blotter:
+            for order in blotter.live_orders:
```
Comment (liampauling) [inline flumine/backtest/backtest.py:97]: Do we move this to live trading as well?

Comment (jsphon) [reply flumine/backtest/backtest.py:97]: What does this mean?

Comment (liampauling) [reply flumine/backtest/backtest.py:97]: As in add it [here](https://github.com/liampauling/flumine/blob/master/flumine/baseflumine.py#L179) as well

**Pull request merged**